UITests/TransferMethod/AddTransferMethodBankAccountTests.swift

```diff
@@ -134,6 +216,27 @@ class AddTransferMethodTests: BaseTests {
         XCTAssertTrue(selectTransferMethodType.navigationBar.exists)
     }
 
+    func testAddTransferMethod_verifyRoutingNumberIsNotEditable() {
+        mockServer.setUpGraphQLBankAccountWithNotEditableField()
```

Comment:
This doesn't work, you need to setup the mocked response before the test starts.

Reply:
Fixed